sphinx/texinputs/sphinx.sty

```diff
@@ -329,6 +329,8 @@
        \discretionary{\copy\Sphinxvisiblespacebox}{\Sphinxafterbreak}
                      {\kern\fontdimen2\font}%
        }%
+  % go around fancyvrb's check of @currenvir (for case of minipage below)
+  \renewcommand*{\VerbatimEnvironment}{\xdef\FV@EnvironName{Verbatim}}%
```

Comment:
this should be rather `\gdef` as there is no expansion to be done here by `\xdef`.